src/main/java/nl/stil4m/mollie/domain/CreatePayment.java

```diff
@@ -7,14 +7,19 @@
     private final Double amount;
     private final String description;
     private final String redirectUrl;
+    private final String webhookUrl;
```

Comment:
Okay, but you do not include the Guava Jackson module, so the getter will need to return a `String` anyway.